go/api/base_client.go

```diff
@@ -512,3 +512,31 @@ func (client *baseClient) RPush(key string, elements []string) (Result[int64], e
 
 	return handleLongResponse(result)
 }
+
+func (client *baseClient) Ping() (string, error) {
```

Comment:
I think we should return Result[string] from all the methods to keep it consistent and pass the nil string info if an error has occurred. Can you please address this in another PR?